myRental/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "test": "mocha --exit"
+    "test": "mocha -t 10000 --exit"
```

Comment:
What is causing the tests to be so slow that we need a 10 second timeout delay?

Reply:
I am not sure, it may just be my laptop. Ten seconds might be overkill, I am sure if finishes sooner than that. I am sure running it on a faster system not bogged down could do it a lot quicker.